10-angular/custom_directives/readme.md

```diff
@@ -536,7 +545,7 @@ Put your partial in a file called `views/grumbles/_grumble_save.html`.
 
```

Comment:
line 540 above, data-grumble should be `GrumbleShowViewModel.grumble` so it's an instance of GrumbleFactory